packages/core/src/theme/theme.ts

```diff
@@ -1,18 +1,22 @@
+import { DefaultTheme } from 'styled-components'
 import { createTextStyles, createColorStyles } from '../utils'
 
-const createMediaQuery = (n) => `@media screen and (min-width:${n})`
+const createMediaQuery = (n: string | number) =>
+  `@media screen and (min-width:${n})`
 
-const addAliases = (arr, aliases) =>
+const addAliases = (arr, aliases: string[]) =>
   aliases.forEach((key, i) =>
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
     Object.defineProperty(arr, key, {
       enumerable: false,
       get() {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return,@typescript-eslint/no-unsafe-member-access
         return this[i]
       },
     })
   )
 
-export const breakpoints = [32, 40, 48, 64, 80].map((n) => n + 'em')
+export const breakpoints = [32, 40, 48, 64, 80].map((n) => `${n}`)
```

Comment:
I think this is the root cause
```suggestion
export const breakpoints = [32, 40, 48, 64, 80].map((n) => `${n}em`)
```